### DISCUSSION TITLE
Use Windows fonts in Latex?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#4017] [Q&A]
Is it possible to configure default_template.tex to allow use of Windows fonts when using the latex \text{} command on Windows 10?

I am using MikTex with xetex installed. I have added:

\usepackage{fontspec}
\setmainfont{Lucida Sans}

to default_template.tex and I have changed the pdf generation command to:

xelatex -halt-on-error -interaction=nonstopmode '{}'

However, if I try to use \text{} from the Latex box, the "OK" button never becomes active. I'm failry clueless about Latex though, so maybe I'm doing something very wrong somewhere. 

Any suggestions?

**Comment by rolandlo** [comment]
In PR #2809 you can directly check the Command Output produced from your tex generator. Studying that output would help to find the problem. So if you know how to compile a PR that would be a good start. Otherwise, you can also get the output within the temporary folder (search it for the `tex.log` file that has been generated when you entered the LaTeX formula).

**Reply by NelkonParker** [reply]
Well, I don't  have a tex.log file, but there is a xelatex.log file, which contains only the following which seems to be of any significance:

2022-05-10 10:47:53,069+0100 INFO  xelatex - starting with command line: "C:\Program Files\MiKTeX 2.9\miktex\bin\x64\\xelatex.exe" -halt-on-error -interaction=nonstopmode C:\Users\Stephen\AppData\Local\Temp\xournalpp-7940\tex\tex.tex
2022-05-10 10:47:53,075+0100 INFO  xelatex - allowing known shell commands
2022-05-10 10:47:53,075+0100 INFO  xelatex - enabling input (output) from (to) processes
2022-05-10 10:47:53,509+0100 INFO  xelatex - finishing with exit code 1

The exit code 1 is going to be an error code, I guess, but there's no more information present about any specific problem. Is there anywhere else that I can look to get more detailed information?

**Reply by Technius** [reply]
Could you try generating another `tex.tex` file from Xournal++ and then running the xelatex command manually?

**Reply by NelkonParker** [reply]
I'm not at all sure how to generate a tex.tex file from Xournal++ - if you can tell me how to do that, I will give it a go.

**Reply by Technius** [reply]
One should be generated if you try using the Latex tool. Alternatively, you can try using the "Test configuration" button in the Latex tab in Preferences.
